test_scripts/RC/InteriorVehicleData_cache/002_GetInteriorVD_without_subscribe_2_requests.lua

```diff
@@ -0,0 +1,39 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0178-GetInteriorVehicleData.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case
+-- 1. Mobile app requested GetInteriorVD(module_1, without subscribe parameter) and HMI sent GetInteriorVD to HMI
+-- 2. Mobile app  GetInteriorVD(module_1, without subscribe parameter) request
+-- SDL must
+-- 1. send GetInteriorVD(module_1, without subscribe parameter) request to HMI
```

Comment:
to be consistent with other cases, there are 2 steps, there shall be 2 expected outputs.
missing 
2. send GetInteriorVD(module_1, without subscribe parameter) request to HMI